include/gvizdoom/DoomGame.hpp

```diff
@@ -41,16 +42,17 @@ class DoomGame {
     int getScreenHeight() const;
     const uint8_t* getPixelsBGRA() const;
     float* getPixelsDepth() const;
+    GameStateContainer& getGameState();
```

Comment:
It's kind of pointless to return mutable reference from a getter - why not just make the member public then? Of course in this case it's necessary since `GameState::get()` is not const-specified. Come to think of it, maybe in this case the best solution would be to wrap the `GameState::get()` like this:
```
template <GameState T_GameState>
const typename GameStateInfo<T_GameState>::Type& getGameState()
{
    return _gameState.get<T_GameState>();
}
```